modules/rntuple.mjs

```diff
@@ -110,6 +110,64 @@ class RBufferReader {
 
 }
 
+const ENTupleColumnType = {
+  kUnknown: 0,
+  kIndex64: 1,
+  kIndex32: 2,
+  kSwitch: 3,
+  kByte: 4,
+  kChar: 5,
+  kBit: 6,
+  kReal64: 13,
+  kReal32: 14,
+  kReal16: 15,
+  kInt64: 16,
+  kUInt64: 17,
+  kInt32: 18,
+  kUInt32: 19,
+  kInt16: 20,
+  kUInt16: 21,
+  kInt8: 22,
+  kUInt8: 23,
+  kSplitIndex64: 24,
+  kSplitIndex32: 25,
+  kSplitReal64: 26,
+  kSplitReal32: 27,
+  kSplitInt64: 28,
+  kSplitUInt64: 29,
+  kSplitInt32: 30,
+  kSplitUInt32: 31,
+  kSplitInt16: 32,
+  kSplitUInt16: 33,
+  kReal32Trunc: 34,
+  kReal32Quant: 35,
+  kMax: 36
+};
+
+// Determine byte size per value based on column type
+function getTypeByteSize(coltype) {
+    switch (coltype) {
+        case ENTupleColumnType.kReal64:
```

Comment:
I'm not sure this function can be very helpful later because some types have a size that's not an integer number of bits (e.g. Bit is 1 bit in size, Real32Trunc and Real32Quant have a variable bit length, etc).
For now it's fine to keep, but keep that in mind.